extensions/interactions/FractionInput/directives/fraction-input-validation.service.ts

```diff
@@ -109,8 +109,9 @@ export class FractionInputValidationService {
     var ranges = [];
     var matchedDenominators = [];
 
+    // Note: rules do not have a set order.
```

Comment:
Perhaps this comment should go on the line below, and should be rephrased to "rules within an answer group".

Reply:
Removed this comment as there are more detailed comments below where it is more relevant.